versions/run.ver

```diff
@@ -47,7 +47,7 @@ export cmc_ver=v1.2
 export fnmoc_ver=v1.1
 export gefs_ver=v12.3
 export gfs_ver=v16.3
-export glwu_ver=v1.1
+export glwu_ver=v2.0
```

Comment:
Is glwu v2.0 in operations right now?

@SamiraArdani-NOAA @malloryprow

Reply:
@AliciaBentley-NOAA, Yes. GLWU v2.0 is in operation.